samples/client/petstore/python/tests/test_pet_api.py

```diff
@@ -13,6 +13,7 @@
 
 import os
 import unittest
+import pprint
```

Comment:
If we only need this for writing to disk and we will remove the disk writing below, then please delete this too

Reply:
I am removing, this was only for testing purpose.